huntsman/utils/pyro/__init__.py

```diff
@@ -128,7 +128,10 @@ def run_camera_server(ignore_local, unmount_sshfs=True):
     mountpoint = sshfs_mount.mount_sshfs()
     
     Pyro4.config.SERVERTYPE = "multiplex"
-    config = load_config(config_files=['pyro_camera.yaml'], ignore_local=ignore_local)
```

Comment:
it might be handy to keep the functionality of loading a specific yaml file instead. Indeed, if you merge this now the system wouldn't work at all.

Let's default to using the local file for now (so `use_server=False`), until we get the config_server online.